lib/megaco/test/megaco_test_lib.hrl

```diff
@@ -151,3 +154,12 @@
 
 -define(MEGACO_TRACE(C, D),    ?LIB:megaco_trace((C), (D))).
 -define(ENABLE_TRACE(C, L, D), ?LIB:enable_trace((C), (L), (D))).
+
+%% 'BENCH_SUITE' needs to be defined by the module using this macro
+-define(BENCH_EVENT(__N__, __V__),
+        #event{name = benchmark_data,
+               data = [{suite, ?BENCH_SUITE},
```

Comment:
is it worth to have ?BENCH_SUITE at all?
in socket you just use `{suite, atom_to_list(?MODULE)}` ... not good enough? why?